app.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "kepler",
+  "buildpacks": [{
+    "url": "https://github.com/dulaccc/heroku-buildpack-geodjango.git#1.1"
```

Comment:
some something like this might do it:
`heroku buildpacks:set https://github.com/dulaccc/heroku-buildpack-geodjango.git#1.1`